api.c

```diff
@@ -41,9 +41,13 @@ int api_bindlisten(const char *api_socket)
 struct api_hostfwd {
     int id;
     int is_udp;
-    struct in_addr host_addr;
+    int is_ipv4;
+    int is_ipv6;
```

Comment:
We shouldn’t need is_ipv4.
Checking !is_ipv6 should be enough.

Reply:
We can pass tcp6/4 as proto to explicitly just enable either.
If we pass tcp as proto, ipv6 is enabled and the host address
is either 0.0.0.0 or ::0, both are enabled.